posts/job_descriptions.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Voting on Job Descriptions
+author: mike
+date: "2015-06-11"
+description: Voting on job descriptions
+tags: ["recruitment","sweets" ]
+categories: ["development", "devops"]
+---
+
+![Voting on Job Descriptions](/assets/media/job_voting/voting.jpg)
+
+
+Like most small companies we have trouble [@MyDrive](https://twitter.com/_mydrive) in attracting the best
+engineering staff when we're recruiting. We're not Google or Apple, with a huge
+global presence, most developers have probably never heard of us. The current
+job market for developers is really competitive, there aren't that many great
+developers out there, most of them are already in work and everyone is competing
+for them.
+
+We've recently been recruiting for an iOS developer role, to lead the development
+of our telematics apps. I think is actually a really interesting, fun role, which
+will involve lots of really quite difficult tasks, really stretching the technology.
+
+Our challenge is to make sure that we can get that across in our job adverts.
+
+Our process includes a couple of [coding tests](https://github.com/mydrive/code-tests). One of which
+candidates can do on their own at home, and a much smaller one which we do in
+house, just to verify what we've seen on the external test.
+
+A few candidates have responded and taken the test and it gives a
+good view of where someone is in their development. It also gives you a real
+example to discuss with a candidate about why they made certain decisions, what
+their thought processes were, and what they'd do differently next time. We haven't
+found the right person yet, but it's been really worthwhile.
+
+Some recruiters have told us that people are put off from applying because there
+is a more challenging process including a test, and hopefully that's just filtering
+out people who aren't suitable for the role, but we could be putting off some
+candidates who really would be great because we've increased the barrier to apply.
+
+All of this caused us to review our Job Descriptions. We quickly realised
+that just tweaking wording maybe wasn't radical enough. There was also a feeling
+that one of the best things about working at [@MyDrive](https://twitter.com/_mydrive) is the working culture
+which wasn't well reflected in the job descriptions.
```

Comment:
Should this be: "which wasn't well reflected in the job description." without the 's'